frontend/src/app/shared/components/attachments/attachments.component.sass

```diff
@@ -0,0 +1,85 @@
+@import "helpers"
+
+.op-attachments
+  display: flex
+  flex-direction: column
+  position: relative
+
+  &--actions
+    @include spot-body-small
+
+  &--drop-box
+    @include unset-button-styles
+    display: flex
+    justify-content: center
+    align-items: center
+    width: 100%
+    margin-top: $spot-spacing-0_75
+    border: 2px dashed var(--light-gray)
+    border-radius: 2px
+    padding: 20px
+    cursor: pointer
+    color: var(--light-gray)
+    background: rgba($spot-color-basic-white, 0.9)
+
+    &_float
+      display: none
+      position: absolute
+      top: 0
+      left: 0
+      height: 100%
+      width: 100%
+      z-index: 1
+      background: transparent
+      border: 0
+
+      &::after,
+      &::before
+        display: block
+        content: ''
+        position: absolute
+
+      &::before
+        top: -1 * $spot-spacing-0_75
+        height: calc(100% + #{$spot-spacing-1_5})
+        left: 0
+        width: 100%
+
+        left: 0
+        width: 100%
+
+        background: rgba($spot-color-basic-white, 0.9)
+        z-index: -2
+
+      &::after
+        z-index: -1
+        top: 0
+        height: 100%
+        left: $spot-spacing-0_75
+        width: calc(100% - #{$spot-spacing-1_5})
+        border: 2px dashed var(--light-gray)
```

Comment:
why not using a spot token here? what's that here, `$spot-color-basic-gray-5`? Same for the `-text` in line 74.